test/Microsoft.Dnx.Testing.CommonUtils/TestUtils.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.Versioning;
+using Microsoft.Dnx.Runtime;
+using Microsoft.Dnx.Runtime.Helpers;
+using System.Runtime.CompilerServices;
```

Comment:
Just to check. I see other files have all Microsoft.Dnx.* after System.* is this the convention?